gateway/cli/src/main.rs

```diff
@@ -30,6 +30,8 @@ pub enum Commands {
     VersionHash,
     /// Display high-level information about the Gateway
     Info,
+    /// Display config information about the Gateways federation
+    Config,
```

Comment:
Imo this should be a per-federation command, i.e. takes the federation id as argument.

Reply:
I wonder which is better. This or `--federation-id` acting as a filter, and lack of it meaning "all".

Reply:
Why do we need the "all" version of the command?

Reply:
I don't know. If you're in one federation only you can get the config without specifying the id. Or if you're writing a script you could loop over all in one go (one `| jq ...`). Not very strong reasons, but also doesn't change anything for someone that just wants one config.